src/components/video.tsx

```diff
@@ -21,19 +26,19 @@ function Video({ youtube, src = format(youtube), caption }: VideoProps) {
           paddingBottom: "56.25%;",
           paddingTop: "25px",
           height: 0,
-          marginBottom: caption ? "0.5em" : 0
+          marginBottom: caption ? "0.5em" : 0,
         }}
       >
         <iframe
-          src={src}
-          title={caption || youtube ? "YouTube Video" : "Videp"}
+          src={youtube ? formatYoutube(youtube) : formatVimeo(vimeo)}
+          title={caption || youtube ? "YouTube Video" : "Video"}
```

Comment:
Let’s default to Video, you can do `caption ?? "Video"`

Reply:
Nullish coalescing doesn't seem to work, I think we might need to upgrade the typescript version on another PR, for now I think good old `caption || "Video"` might work